Source/Renderer/TimerQuery.js

```diff
@@ -0,0 +1,64 @@
+define([
+        '../Core/WebGLConstants',
+        '../Core/oneTimeWarning'
+    ], function(
+        WebGLConstants,
+        oneTimeWarning) {
+    'use strict';
+
+    /**
+     * Represents a timer query.
+     *
+     *
+     * @param {Object} frameState The current FrameState.
+     * @param {Function} callback The callback to execute after the frame time is measured. Takes the frame time in millisecond as a parameter.
+     * @private
+     */
+    function TimerQuery(frameState, callback) {
+        var context = frameState.context;
+        this._context = context;
+
+        if (!processingSupported(context)) {
```

Comment:
I'm pretty sure this should throw a `DeveloperError` like we do elsewhere when an extension is not available, it is up to the developer to ensure they are only using this when it is available.